utils/eth.go

```
@@ -26,8 +30,12 @@ func GetWEIFromETH(val *float64) *big.Int {
 }
 
 func GetPrivateKey(key string) (*ecdsa.PrivateKey, error) {
-	if strings.HasPrefix(key, "0x") {
-		key = key[2:]
+	decryptedKey, err := Decrypt([]byte(config.Opts.PrivateKeySecret), key)
+	if err != nil {
+		log.Fatal("Unable to decrypt Private Key!", err)
```

ich wür mal im code nach log.Fatal sueche, das wot mer vermuetlich nie

---

hmm naja wenns geld verlore goh bini eher en fan vo fail fast